perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -149,8 +160,24 @@ def _IsReady(self):
     Returns:
       True if the resource was ready in time, False if the wait timed out.
     """
+    cmd = util.GcloudCommand(self, 'sql', 'instances', 'describe',
+                             self.instance_id)
+    stdout, _, _ = cmd.Issue()
+    json_output = json.loads(stdout)
+    is_ready = (json_output[0]['state']
+                == 'RUNNABLE')
+    if not is_ready:
+      return False
+    self.endpoint = self._ParseEndpoint(json_output)
+    self.port = self.MYSQL_DEFAULT_PORT
     return True
 
+  def _ParseType(self, describe_instance_json):
```

Comment:
This isn't used anywhere - what is its purpose? Also the naming and intention is unclear, because it isn't parsing the type - it is returning whether or not the instance is a sql instance.